cfpb/complaint
# Update readme and changelog

Updated the readme and changelog to reflect all the recent changes.

## Removals

- Removed `screenshot-complaint.png` since that screen is no longer powered by this app 

## Changes

- Updated the changelog
- Updated the CCDB screenshot
- Updated the top-line description of the app

## Testing

- Make sure the updated text makes sense
- Make sure all the changelog links work

## Todos

- There's probably more readme cleanup to be done later, like removing any steps that pertain only to the old `/complaint` pages and not the CCDB. I left all that in there for now, though.
